hooks/reducers/mqlQueryReducer.ts

```diff
@@ -121,6 +123,7 @@ const mqlQueryReducer = <CreateQueryDataType extends unknown, FetchQueryDataType
       }
       return {
         ...state,
+        queryId: null,
```

Comment:
In the case of a totally fresh action of this type, this will already be `null`, but if `action.doRefetchMqlQuery === true`, `state.queryId` will not be `null`, but we need it to be.